### PR TITLE
Updating to pylexibank2.1

### PR DESCRIPTION
**Pull request by tresoldi**
This PR updates the dataset to pylexibank2.1 (glottolog 4.0, concepticon 2.2.0, clts 1.4.1), as needed for the MIT chapter

### CODE REVIEW

File: cldf/forms.csv [hunk 1]
```diff
@@ -866,7 +855,8 @@ Dikhyang-Lieberherr-2017-100-58-1,,Dikhyang,Lieberherr-2017-100-58,siam,siam,s i
 Dikhyang-Lieberherr-2017-100-59-1,,Dikhyang,Lieberherr-2017-100-59,ə-bɛŋ,ə-bɛŋ,ə + b ɛ ŋ,,Lieberherr2017,11701,
 Dikhyang-Lieberherr-2017-100-60-1,,Dikhyang,Lieberherr-2017-100-60,bu-rui,bu-rui,b u + r ui,,Lieberherr2017,11901,
 Dikhyang-Lieberherr-2017-100-61-1,,Dikhyang,Lieberherr-2017-100-61,raguŋ ju-ŋã,raguŋ ju-ŋã,r a g u ŋ + j u + ŋ ã,,Lieberherr2017,12103,
-Dikhyang-Lieberherr-2017-100-62-1,,Dikhyang,Lieberherr-2017-100-62,ə-vɔ̃,ə-vɔ̃,ə + v ɔ̃,,Lieberherr2017,12302,
+Dikhyang-Lieberherr-2017-100-62-1,,Dikhyang,Lieberherr-2017-100-62,"ə-vɔ̃
+",ə-vɔ̃,ə + v ɔ̃,,Lieberherr2017,12302,
```
Comment (SimonGreenhill) [inline cldf/forms.csv]: something weird here?

File: lexibank_lieberherrkhobwa.py [hunk 2]
```diff
@@ -25,6 +27,13 @@ class Dataset(BaseDataset):
     id = "lieberherrkhobwa"
     dir = Path(__file__).parent
     language_class = KBLanguage
+    form_spec = FormSpec(
+            separators = "~/,;⪤",
+            missing_data = (
+                "NA",
+                ),
+                            first_form_only=True
+            )
```
Comment (SimonGreenhill) [inline lexibank_lieberherrkhobwa.py]: can we fix the weird spacing here please?

File: lexibank_lieberherrkhobwa.py [hunk 3]
```diff
@@ -36,92 +45,64 @@ def cmd_download(self, **kw):
...
-                                )
+    def cmd_makecldf(self, args):
+        # Add bibliographic sources
+        args.writer.add_sources()
+
+        # Read raw concept data and add to dataset; at the same time,
+        # build a map between the concept index as used in data and the
+        # concept id in the dataset
+        concept_lookup = {}
+        for cid, concept in enumerate(self.conceptlists[0].concepts.values()):
```
Comment (LinguList) [inline lexibank_lieberherrkhobwa.py]: Sorry for annoying you here, but can I ask you to stick to our general ID handling for concepts, as we practiced it in the other lists?

```python
concept_lookup[1+(cid*2)] = concept.id.split('-')[-1]+'_'+slug(concept.english)
```

File: cldf/forms.csv (hunk 1 above)
Comment (tresoldi) [reply cldf/forms.csv]: Weird indeed, I must have missed something when building the FormSpec.

Comment (tresoldi) [reply cldf/forms.csv]: The issue was that, in previous version, the replacement was performed on the `value` -- `FormSpec` only takes care, as expected, of `form`s.

**Comment by tresoldi** [comment]
Updating, please check.

**Comment by tresoldi** [comment]
Great! Merging and moving to the tutorial.

**Pull request merged**